lib/kubeclient.rb

```diff
@@ -40,7 +40,7 @@ def rest_client
           ssl_client_cert:  @ssl_options[:client_cert],
           ssl_client_key:   @ssl_options[:client_key]
         }
-        endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' \
+        endpoint_with_version = @api_endpoint.merge(@api_endpoint.path \
```

Comment:
In fact, I would use interpolation here as well.  `+` on String is generally avoided.